src/client/app/left/date-picker/date-picker.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="row date-picker">
+  <br/>
+  <div class="col-md-3">Please select the date: </div>
+  <div class="col-md-9">
+    <input type="date" min="2011-01-01" max={{max}} value="2017-02-27" (change)="datePickerUpdated($event)"/>
```

Comment:
Just tested this on Safari and no date picker — just a text field — is it worth trying a jQuery fallback do you think?